curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/add-key-value-pairs-to-javascript-objects.md

```diff
@@ -86,7 +86,8 @@ The definition of the `foods` object should not be changed.
 ```js
 assert(
   code.search(/let foods/) === -1 &&
-  code.search(/const\s+foods\s*=\s*{[\s\S]*}(?![^;]*\b(?:bananas|grapes|strawberries)\s*:\s*[^:])/) !== -1
+  code.search(/const\s+foods\s*=\s*{\s*apples:\s*25,\s*oranges:\s*32,\s*plums:\s*28\s*};/
+) !== -1
 );
 ```
```

Comment:
the test seems to work, when tested through GitPod on my machine. 

One tiny concern: if the user adds a trailing comma after the `plums` property in the foods object literal, this test fails. 

Perhaps that is okay, because the test explanation says that the foods object literal definition should not be changed.

Otherwise, I can try to find a way to accept an optional trailing comma in the regex.